tools/Config.mk

```diff
@@ -393,11 +393,18 @@ endef
 # ARCHIVE - Same as above, but ensure the archive is
 # created from scratch
 
+ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+define ARCHIVE
+	$(Q) $(RM) $1
+	$(Q) $(AR) $1 $(2)
+endef
+else
 define ARCHIVE
 	@echo "AR (create): ${shell basename $(1)} $(2)"
```

Comment:
can we change to:
@echo "AR(create): $(1) $(2)"
to avoid the duplication.

Reply:
BTW, why line 389 doesn't hit the similar issue on Windows?

Reply:
L389:
`define ARCHIVE_ADD
	@echo "AR (add): ${shell basename $(1)} $(2)"
	$(Q) $(AR) $1 $(2)
endef`
`ARCHIVE_ADD ` was used to archive file to `$1` one by one, so `$(2)` didn't contain too many object, and will not give too long input line error

Reply:
> can we change to:
> ```
> @echo "AR(create): $(1) $(2)"
> ```
> to avoid the duplication.

I tried, but failed! the key problem here is echo too many character in Windows environment,  even `@echo "AR (create): $(2)"` give same error by test.
how about remove `@echo "AR (create): ${shell basename $(1)} $(2)"` as well in Linux to avoid the duplication?